.expeditor/config.yml

```diff
@@ -50,28 +51,32 @@ merge_actions:
   - built_in:bump_version:
       post_commit: false
       ignore_labels:
-        - "Version: Skip Bump"
+        - "Expeditor: Skip Version Bump"
         - "Expeditor: Skip All"
         - "Expeditor: ACC Only"
+        - "Aspect: Documentation"
   - built_in:update_changelog:
       post_commit: false
       ignore_labels:
-        - "Changelog: Skip Update"
+        - "Expeditor: Skip Changelog"
         - "Expeditor: Skip All"
         - "Expeditor: ACC Only"
+        - "Aspect: Documentation"
   - trigger_pipeline:omnibus/release:
       post_commit: true
       ignore_labels:
-        - "Omnibus: Skip Build"
+        - "Expeditor: Skip Omnibus"
         - "Expeditor: Skip All"
         - "Expeditor: ACC Only"
+        - "Aspect: Documentation"
       only_if: built_in:bump_version
   - trigger_pipeline:habitat/build:
       post_commit: true
       ignore_labels:
-        - "Omnibus: Skip Build"
+        - "Expeditor: Skip Habitat"
         - "Expeditor: Skip All"
```

Comment:
I would have expected Skip All to handle Skipping Habitat

Reply:
Hab should be skipped if someone uses Skip All or Skip Habitat.